### ISSUE TITLE
cli 0.9.0 fails: Cannot find module '..'

### ISSUE THREAD

**Issue by qqiao** [Polymer/polymer-cli#210]
node version: 6.2.0
Polymer-cli version: 0.9.0

Error message:

```
$ polymer init
module.js:442
    throw err;
    ^

Error: Cannot find module '..'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at /home/q/.nvm/versions/node/v6.2.0/lib/node_modules/polymer-cli/bin/polymer.js:19:36
    at /home/q/.nvm/versions/node/v6.2.0/lib/node_modules/polymer-cli/node_modules/resolve/lib/async.js:46:14
    at process (/home/q/.nvm/versions/node/v6.2.0/lib/node_modules/polymer-cli/node_modules/resolve/lib/async.js:173:43)
    at ondir (/home/q/.nvm/versions/node/v6.2.0/lib/node_modules/polymer-cli/node_modules/resolve/lib/async.js:188:17)
    at load (/home/q/.nvm/versions/node/v6.2.0/lib/node_modules/polymer-cli/node_modules/resolve/lib/async.js:69:43)
    at onex (/home/q/.nvm/versions/node/v6.2.0/lib/node_modules/polymer-cli/node_modules/resolve/lib/async.js:92:31)
```

**Comment by yordis** [comment]
@justinfagnani @FredKSchott Yeah, the 0.9 version introduce this one :(

**Comment by qqiao** [comment]
Same on node v4.4.5

**Comment by dannybrian** [comment]
Also on 4.4.5. It's not just init, it's polymer cli.

**Comment by benjamincobb** [comment]
I also have this problem. Introduced with 0.9.0.

**Comment by justinfagnani** [comment]
I'll look at this ASAP

**Comment by taviLaies** [comment]
+1

**Comment by thomasweitzel** [comment]
After $ npm -g install polymer-cli there's no "lib" directory anymore ...

**Comment by FredKSchott** [comment]
This is caused by https://github.com/Polymer/polymer-cli/issues/192

The `depcheck` task called inside of `build-all` is failing, causing the actually build task to never be run.

**Comment by FredKSchott** [comment]
Fix submitted here: https://github.com/Polymer/polymer-cli/pull/211

**Comment by FredKSchott** [comment]
Proper fix submitted and reviewed here: #212 
We're just waiting for tests to run/pass and then we'll cut a new version, hang tight

**Comment by justinfagnani** [comment]
Should be fixed in 0.10.0 now. Sorry for the problems!
